tutorials/basics/4_batch.py

```diff
@@ -0,0 +1,283 @@
+"""
+.. currentmodule:: dgl
+
+Batched Graph Classification with DGL
+=====================================
+
+**Author**: `Mufei Li <https://github.com/mufeili>`_,
+`Minjie Wang <https://jermainewang.github.io/>`_
+
+In this tutorial, we will learn how to perform batched graph classification
```

Comment:
Could you add a small motivation paragraph in the very beginning, about why graph classification is useful? Could also cite some papers.

Reply:
Better?